core/src/com/unciv/logic/automation/UnitAutomation.kt

```diff
@@ -469,16 +497,43 @@ object UnitAutomation {
         unit.civInfo.addNotification("[${unit.displayName()}] finished exploring.", unit.currentTile.position, unit.name, "OtherIcons/Sleep")
         unit.action = null
     }
-
+    
+    /** Returns whether the civilian spends its turn hiding and not moving */
+    fun tryRunAwayIfNeccessary(unit: MapUnit): Boolean {        
+        // This is a little 'Bugblatter Beast of Traal': Run if we can attack an enemy
```

Comment:
"... A creature so mondbogglingly stupid..."

Reply:
Not my comment - just copied and pasted it from somewhere :)

Reply:
Finally someone noticed. Had to add those to the dictionary, too, so Marvin Studio stopped squiggying them!

Reply:
But honestly, that should have been a to-do: The check _is_ the wrong way round, at least where I put the comment.